public/js/user.js

```diff
@@ -326,7 +326,7 @@ function saveMarker(e) {
 }
 
 function updateMarker(id) {
-    var json = JSON.stringify({ });
+    var json = JSON.stringify({ created_by: currentUser });
```

Comment:
I was mostly testing as admin so I didn't spot this issue before. Since PATCH checks whether the user in the session is the user who owns the marker, we'll have to pass this data in.

Reply:
Ah. The fix to this is to check the database. Otherwise, any non-admin authenticated user could takeover ownership of any marker simply by reading it from the API, setting `created_by` to their username, and PATCH the marker. With this code change, you're making it easier, since all they need to do is invoke this method to takeover  ownership of a marker.